catalog/imaging/imaging-nitf-api/pom.xml

```diff
@@ -27,6 +27,10 @@
     <name>Alliance :: Imaging :: Nitf :: API</name>
     <packaging>bundle</packaging>
 
+    <properties>
```

Comment:
I think it's better to keep the properties scoped as privately as we can without duplicating them.  As far as I know the usng packages aren't used outside of the imaging-app bundles, so I'd like to leave this here.  I am removing the duplicate one.